### PR TITLE
added py-multiqc and dependencies

### PR DESCRIPTION
**Pull request by JusticeForMikeBrown**
Here's the near-useless [documentation](http://multiqc.info/docs/) 

```
user@node103[~/repos/spack]$ spack load -r py-multiqc                                                                                                                                                                                      
user@node103[~/repos/spack]$ multiqc c
Usage: multiqc [OPTIONS] <analysis directory>

Error: Invalid value for "analysis_dir": Path "c" does not exist.

This is MultiQC v1.0 (8939f77)

For more help, run 'multiqc --help' or visit http://multiqc.info

```

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-multiqc/package.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    homepage = "https://multiqc.info"
+    url      = "https://github.com/ewels/MultiQC/tarball/v1.0"
+
+    version('1.0', '27afc19327105818e39b385211b27694')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-click', type='run')
+    depends_on('py-jinja2', type='run')
+    depends_on('py-lzstring', type='run')
+    depends_on('py-future', type='run')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-multiqc/package.py]: `py-future@0.14.1:`

File: var/spack/repos/builtin/packages/py-multiqc/package.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    for a large number of common bioinformatics tools."""
+
+    homepage = "https://multiqc.info"
+    url      = "https://github.com/ewels/MultiQC/tarball/v1.0"
+
+    version('1.0', '27afc19327105818e39b385211b27694')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-click', type='run')
+    depends_on('py-jinja2', type='run')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-multiqc/package.py]: `py-jinja2@2.9:`

File: var/spack/repos/builtin/packages/py-multiqc/package.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    depends_on('py-lzstring', type='run')
+    depends_on('py-future', type='run')
+    depends_on('py-spectra', type='run')
+    depends_on('py-colormath', type='run')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-multiqc/package.py]: Missing dependencies on `py-matplotlib`, `py-numpy`, `py-pyyaml`, and `py-simplejson`.

`py-colormath` is not a dependency.

All of these dependencies aside from `py-setuptools` should be build/run.

File: var/spack/repos/builtin/packages/py-spectra/package.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PySpectra(PythonPackage):
+    """Color scales and color conversion made easy for Python."""
+
+    homepage = "https://pypi.python.org/pypi/spectra/0.0.8"
+    url      = "https://files.pythonhosted.org/packages/31/67/f73242e5ac4fb772e8c901dafac5b7195f8dc179e9a0665c85b3e8512d2c/spectra-0.0.8.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-spectra/package.py]: Please download from PyPI:
https://pypi.io/packages/source/s/spectra/spectra-0.0.8.tar.gz

File: var/spack/repos/builtin/packages/py-spectra/package.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+    version('0.0.8', '83020b29e584389f24c7720f38f0136c')
+
+    depends_on('py-setuptools', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-spectra/package.py]: Missing dependency on `py-colormath`.

**Comment by JusticeForMikeBrown** [comment]
Thanks again @adamjstewart!

File: var/spack/repos/builtin/packages/py-spectra/package.py (hunk 4 above)
Comment (JusticeForMikeBrown) [inline var/spack/repos/builtin/packages/py-spectra/package.py]: Fixed

File: var/spack/repos/builtin/packages/py-spectra/package.py (hunk 5 above)
Comment (JusticeForMikeBrown) [inline var/spack/repos/builtin/packages/py-spectra/package.py]: Fixed

**Comment by JusticeForMikeBrown** [comment]
@adamjstewart can we merge, please?

**Comment by adamjstewart** [comment]
@JusticeForMikeBrown Sorry, I have a tutorial to lead today and tomorrow so I'm not going to be able to review your PRs anytime soon. If anyone else has a chance to get to this they can merge them.

**Comment by JusticeForMikeBrown** [comment]
@adamjstewart thanks for letting me know!  have fun!

File: var/spack/repos/builtin/packages/py-multiqc/package.py [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    for a large number of common bioinformatics tools."""
+
+    homepage = "https://multiqc.info"
+    url      = "https://pypi.io/packages/source/m/multiqc/multiqc-1.0.tar.gz"
+
+    version('1.0', '0b7310b3f75595e5be8099fbed2d2515')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-click', type=('build', 'run'))
+    depends_on('py-jinja2', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-multiqc/package.py]: `py-jinja2@2.9:`

File: var/spack/repos/builtin/packages/py-multiqc/package.py [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    depends_on('py-lzstring', type=('build', 'run'))
+    depends_on('py-future', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-multiqc/package.py]: `py-future@0.14.1:`

File: var/spack/repos/builtin/packages/py-multiqc/package.py [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+    depends_on('py-spectra', type=('build', 'run'))
+    depends_on('py-colormath', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-multiqc/package.py]: Missing dependencies on matplotlib, numpy, pyyaml, and simplejson. colormath is not a dependency. See the `setup.py`:
```python
    install_requires = [                                                        
        'click',                                                                
        'future>0.14.0',                                                        
        'lzstring',                                                             
        'jinja2>=2.9',                                                          
        'matplotlib',                                                           
        'numpy',                                                                
        'pyyaml',                                                               
        'simplejson',                                                           
        'spectra',                                                              
    ],              
```

File: var/spack/repos/builtin/packages/py-spectra/package.py (hunk 4 above)
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-spectra/package.py]: Did you forget to push the commit? It doesn't appear to be fixed.

**Comment by adamjstewart** [comment]
Ping @JusticeForMikeBrown

**Comment by JusticeForMikeBrown** [comment]
@adamjstewart 

Sorry, been away from the office then busy upon return.  I believe the issues with this package have been addressed.

Thanks!

File: var/spack/repos/builtin/packages/py-multiqc/package.py [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+    version('1.0', '0b7310b3f75595e5be8099fbed2d2515')
+
+    depends_on('python@2.7:')
+    depends_on('py-setuptools', type='build')
+    depends_on('py-click', type=('build', 'run'))
+    depends_on('py-jinja2@2.9:', type=('build', 'run'))
+    depends_on('py-lzstring', type=('build', 'run'))
+    depends_on('py-future@0.14.1:', type=('build', 'run'))
+    depends_on('py-spectra', type=('build', 'run'))
+    depends_on('py-colormath', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-multiqc/package.py]: I don't think `colormath` is a dependency, is it?

**Comment by JusticeForMikeBrown** [comment]
@adamjstewart Yes, spectra needs it but multiqc does not.

**Pull request merged**